### PR TITLE
add bearer token to getlcuster call

### PR DESCRIPTION
**Pull request by maratoid**

### CODE REVIEW

File: internal/cluster-manager-api/cluster.go [hunk 1]
```diff
@@ -167,3 +169,16 @@ func (s *Server) AdjustClusterNodes(ctx context.Context, in *pb.AdjustClusterMsg
 	}
 	return nil, status.Error(codes.InvalidArgument, InvalidProviderErrorMessage)
 }
+
+// Add bearer token info to GetCluster() call reply
+func (s *Server) setBearerToken(clusterReply *pb.GetClusterReply, err error) (*pb.GetClusterReply, error) {
+	secretName := clusterReply.Cluster.GetName() + "-" + k8sutil.SDSServiceAccountName + "-token"
+	secret, err := k8sutil.GetSecret(secretName, viper.GetString("kubernetes-namespace"))
```
Comment (davidewatson) [inline internal/cluster-manager-api/cluster.go]: Is there only one namespace/token per CMA deployment?

Comment (zachpuck) [reply internal/cluster-manager-api/cluster.go]: No, there is one token per SDSCluster created. it is named after the created cluster. example 
`zachtest01-sds-sa-token`

Comment (davidewatson) [reply internal/cluster-manager-api/cluster.go]: I see, so there is one token per cluster and they are all stored in the same namespace. Ideally, these tokens would be stored in the same namespace as the cluster they provide access for, but due to technical debt not all of the CMA providers handle cluster namespaces the same way. We should fix (and document!) this... :)

**Pull request merged**